scipy/ndimage/interpolation.py

```diff
@@ -211,17 +211,31 @@ def geometric_transform(input, mapping, output_shape=None,
 
     Examples
     --------
-    >>> from scipy import ndimage
+    >>> import numpy as np
+    >>> from scipy.ndimage import geometric_transform
     >>> a = np.arange(12.).reshape((4, 3))
     >>> def shift_func(output_coords):
     ...     return (output_coords[0] - 0.5, output_coords[1] - 0.5)
     ...
-    >>> ndimage.geometric_transform(a, shift_func)
+    >>> geometric_transform(a, shift_func)
     array([[ 0.   ,  0.   ,  0.   ],
            [ 0.   ,  1.362,  2.738],
            [ 0.   ,  4.812,  6.187],
            [ 0.   ,  8.263,  9.637]])
-
+    >>>
```

Comment:
Ah sorry, should have been more clear here. What I meant was break up the examples like this:

```
...
>>> geometric_transform(a, shift_func)
array([[ 0.   ,  0.   ,  0.   ],
           [ 0.   ,  1.362,  2.738],
           [ 0.   ,  4.812,  6.187],
           [ 0.   ,  8.263,  9.637]])

>>> b = [1, 2, 3, 4, 5]
...
```

Then each example will get rendered in a separate gray box. (See e.g. https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.diric.html#scipy.special.diric and the source https://github.com/scipy/scipy/blob/v1.0.0/scipy/special/basic.py#L80.)